### DISCUSSION TITLE
Is there a C# wrapper?

### DISCUSSION THREAD

**Discussion by Grainbox** [pocketbase/pocketbase#3667] [Q&A] [answered]
Hi, i'm about to work with pocketbase in C#, i was wondering if there exists a wrapper to discuss with the pocketbase api in this language? If there is not, is there a documentation for the js/dart sdk so i can make my own?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no official C# wrapper at the moment.

In the past someone mentioned https://github.com/iluvadev/PocketBaseClient but I haven't tried it and I don't know what is supported and whether it works with the most recent PocketBase versions.

For the Web APIs documentation you can check the [Web APIs reference](https://pocketbase.io/docs/api-records/).

For JS or Dart SDK docs, please refer to their repos:
- https://github.com/pocketbase/js-sdk
- https://github.com/pocketbase/dart-sdk

**Reply by Grainbox** [reply]
Okay, thank you, i'll check it!
